client/rest-high-level/src/main/java/org/elasticsearch/client/ccr/PauseFollowResponse.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.ccr;
+
+import org.elasticsearch.client.rollup.AcknowledgedResponse;
+import org.elasticsearch.common.xcontent.ConstructingObjectParser;
+import org.elasticsearch.common.xcontent.XContentParser;
+
+import java.io.IOException;
+
+public final class PauseFollowResponse extends AcknowledgedResponse {
```

Comment:
I reused `AcknowledgedResponse` from rollup. If we are ok with that then maybe we should move it to another package (`o.e.client.common`)?  

Also many APIs have exactly this same response (`acknowledged` field). So maybe we can change AcknowledgedResponse to be a concrete class (but still allowing subclasses to change the field name), so that for this API and others we can directly use `AcknowledgedResponse` class instead of introducing a subclass for each API.

Reply:
https://github.com/elastic/elasticsearch/pull/34623#pullrequestreview-168596221

Id be perfectly happy with you moving it to the `core` package. Re that link and the review in it, I had suggested that exactly to the author. It looks like i forgot to have him remove `abstract` from the class, so feel free to do that too!